rd-net/Lifetimes/Lifetimes/LifetimeDefinition.cs

```diff
@@ -21,7 +19,7 @@ namespace JetBrains.Lifetimes
   /// </summary>
   public class LifetimeDefinition : IDisposable
   {    
-    
+#pragma warning disable 420
```

Comment:
I have no particular objection to that. But I think that code would be more polluted. Hm, Maybe we should just lower the level of this inspecction to warning?

Reply:
Yes, it would make the code exactly four times more polluted, but, at the same time, it would be less prone to the issue this warning tries to warn us about. So, I would lean toward adding smaller sections of `#pragma warning disable 420`.